install.sh

```diff
@@ -13,20 +13,22 @@ sha256_verify ()
 
 deps_install ()
 {
+    common_deps=( \
+        'python-virtualenv' \
+        'curl' \
+        'build-essential' \
+        'automake' \
+        'pkg-config' \
+        'libtool' \
+        'libgmp-dev' )
+
     if [[ ${install_os} == 'debian' ]]; then
         if is_python3; then
-            if deb_deps_install "python-virtualenv curl python3-dev python3-pip build-essential automake pkg-config libtool libgmp-dev"; then
-                return 0
-            else
-                return 1
-            fi
+            deb_deps_install "${common_deps[@]} python3-dev python3-pip"
```

Comment:
might be useful to also install `libsodium` at this step

Reply:
I'd like to keep Debian\Ubuntu OS deps install stuff separated from other stuff that we build from source like `libsodium`.  Easier to manage flow.